src/components/MergeArticleList.tsx

```diff
@@ -7,10 +7,20 @@ const Container = styled.div`
 `
 
 const reads: Array<CardListItem> = [
+   {
+    title: "Technical Aspects of Ethereum Merge",
+    description: "Alchemy",
+    link: "https://www.alchemy.com/overviews/the-ethereum-merge",
+  },
   {
-    title: "Ethmerge",
-    description: "Ethmerge",
-    link: "https://ethmerge.com/",
+    title: "How to Migrate from Ropsten to Goerli",
```

Comment:
I'm not sure this content is right for the further reading. It's very specific to dapp developers and we probably want to keep the further reading a little more generalised.

Reply:
Agreed, I think this link makes more sense to include in the "Dapp and smart contract developers" section.